Kitodo/src/main/java/org/kitodo/production/services/file/FileService.java

```diff
@@ -526,7 +526,7 @@ public boolean canRead(URI uri) {
     public String getFileName(URI uri) {
         String fileNameWithExtension = fileManagementModule.getFileNameWithExtension(uri);
         if (fileNameWithExtension.contains(".")) {
-            return fileNameWithExtension.substring(0, fileNameWithExtension.indexOf('.'));
+            return fileNameWithExtension.substring(0, fileNameWithExtension.lastIndexOf('.'));
```

Comment:
It seems to me, that the intent of the getFileName method (before and after the change) is to get the name of the file without the extension. Could we not just rely on the `getBaseName`-method of the Commons IO-library here?

https://commons.apache.org/proper/commons-io/javadocs/api-2.5/org/apache/commons/io/FilenameUtils.html#getBaseName(java.lang.String)

Reply:
@BartChris : thanks for review!
Yes we can refactor the `getFileName` method and use `getBaseName` method of the Commons IO-library.